### ISSUE TITLE
Meaning of 'upstream' and 'origin' is reversed

### ISSUE THREAD

**Issue by gcapes** [hoytpr/git_beginner#1]
These notes indicate that the user's fork of a SWC repo is called 'upstream'. This is the opposite to the normal naming convention, where SWC would be referred to as the upstream repo (changes flow downstream to the user fork ('origin')).

**Comment by hoytpr** [comment]
Thanks @gcapes. You are an expert and this only illustrates how I am not. I've seen this isuue used both ways and to me the convention is not clear. This may have been based on Max Belkin's advice, which appeared to be using: 
```
git checkout -f gh-pages
git fetch origin
git reset --hard origin/gh-pages
```

to pull the SWC final (most current) product down to your local repos. I couldn't interpret it any other way because I've never seen a way to get my forked GitHub repo to be up-to-date and without a PR-history, **unless I just deleted and re-forked it**.  

In other words... the point of this is to force some attention to these details, and I really appreciate your advice. I'd have to remake all the figure to change this, which I might not have time to do, but maybe I can ad a disclaimer at the beginning that this is NOT the correct naming convention. If you have any suggestions that might help, feel free to send in a PR!

Pete

**Comment by gcapes** [comment]
No worries! I saw this on the SWC mailing list so thought I'd add a couple of ideas.

I'm planning to write something vaguely similar for the Carpentry Connect event in the summer - perhaps you'd be interested to look it over when I've got a first draft?

**Comment by hoytpr** [comment]
Absolutely!

**Comment by gcapes** [comment]
Comments welcome on https://github.com/gcapes/swc-pr-tutorial if you have time to take a look?
